src/Presence/gmail.json

```
@@ -0,0 +1,126 @@
+{
+   "gmail.categoryPrimary": {
+      "message": "Viewing the Primary category",
+      "description": "Displayed when viewing primary emails."
+   },
+   "gmail.categorySocial": {
+      "message": "Viewing the Social category",
+      "description": "Displayed when viewing social emails."
+   },
+   "gmail.categoryUpdates": {
+      "message": "Viewing the Updates category",
+      "description": "Displayed when viewing updates emails."
+   },
+   "gmail.categoryPromotions": {
+      "message": "Viewing the Promotions category",
+      "description": "Displayed when viewing promotions emails."
+   },
+   "gmail.categoryForum": {
+      "message": "Viewing the Forum category",
+      "description": "Displayed when viewing forum emails."
+   },
+   "gmail.inLabel": {
+      "message": "Viewing label",
+      "description": "Displayed when a user is browsing emails in certain label"
```

Assuming this is true
```suggestion
      "description": "Displayed when a user is browsing emails in a certain label. The name of the label will appear below."
```